src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/BookingServiceTest.kt

```diff
@@ -6868,6 +6874,7 @@ class BookingServiceTest {
       )
 
       every { mockUserAccessService.userMayCancelBooking(user, booking) } returns true
+      every { mockFeatureFlagService.getBooleanFlag("cas1-check-delius-withdrawal-status", false) } returns true
```

Comment:
a tiny details, but did you name this `cas1-check-delius-withdrawal-status` because you anticipate the potential need for other Delius tests which affect withdraw-ability? If not, would `cas1-check-delius-arrival-status` be more descriptive?